package.json

```diff
@@ -68,6 +68,43 @@
       }
     ],
     "languageModelTools": [
+            {
+        "name": "list_Topics",
+        "when": "confluent.chatParticipantEnabled",
+        "tags": [
+          "topics"
+        ],
+        "displayName": "List Topics",
+        "modelDescription": "List all available topics in the Kafka cluster. Use this when the user asks about available topics or you want to show them a list of topics.",
+        "canBeReferencedInPrompt": false,
+        "icon": "$(list-selection)",
+        "inputSchema": {
+          "type": "object",
+          "properties": {
+            "kafkaClusterId": {
+              "type": "string",
+              "description": "The ID of the specific Kafka cluster to list topics for. Note: This is distinct from the connection and environment IDs, which will share the same value in LOCAL and DIRECT connection types."
+            },
+            "topicNameSubstring": {
+              "type": "string",
+              "description": "An optional substring to filter the list of topics by name."
+            },
+            "environmentId": {
+              "type": "string",
+              "description": "The ID of the environment containing the Kafka cluster. For CCLOUD connections, this is a distinct environment ID. For LOCAL and DIRECT connections, this will match the connectionId."
+            },
+            "connectionId": {
+              "type": "string",
+              "description": "The ID of the connection to use. Identifies which connection configuration to use when accessing the Kafka cluster."
+            }
```

Comment:
No action needed in this branch, but we'll want to keep an eye on this behavior to see if we also need to add more context to the description(s) for the `get_connections` tool to really emphasize the "environment ID and connection ID are the same for `LOCAL`/`DIRECT`" idea

Reply:
ack